Explorer/Assets/DCL/CommunicationData/URLHelpers/URN.cs

```diff
@@ -31,15 +29,13 @@ public bool IsNullOrEmpty() =>
         public bool IsValid() =>
             !IsNullOrEmpty() && originalUrn.StartsWith("urn");
 
-        public bool Equals(int other) => Equals(other.ToString());
-
         public bool Equals(URN other) =>
-            Equals(other.lowercaseUrn);
+            Equals(other.originalUrn);
 
-        public bool Equals(string other) =>
-            string.Equals(lowercaseUrn, other);
+        private bool Equals(string other) =>
```

Comment:
StringComparison.OrdinalIgnoreCase is really slow this is why I introduced the lowercase cached version for comparison does this have an improvement?

Reply:
ok, brought it back but with Memory and Span approach instead of creating new string.